src/api/glue-api.js

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios'
+// import qs from 'qs'
```

Comment:
Maybe you can remove this unnecessary comment. :hehe:

Reply:
Yes :))